app/build.gradle

```diff
@@ -56,7 +56,7 @@ android {
     unitTests {
       includeAndroidResources = true
       all {
-        maxHeapSize = "1024m"
+        maxHeapSize = "4024m"
```

Comment:
Probably should use 4096 (it's generally conventional to use multiples of '1024').

Reply:
done